src/components/MeetingsComponents/MeetingsForm.js

```diff
@@ -0,0 +1,83 @@
+import React from 'react';
+import { Button, Form, FormGroup, Label, Input, FormText } from 'reactstrap';
+
+class MeetingsForm  extends React.Component {
+    state = {
+        title: '',
+        startTime: '',
+        endTime: '',
+        location: '',
+        description: '',
+        
+    }
+
+    render() {
+        return(
```

Comment:
make sure you use consistent spacing and proper indentations for all the elements in the app.

Reply:
... or at least set up a proper linter

Reply:
what are the linter extensions you use?

Reply:
Ok I got it, my formatter was not working